mozilla_django_oidc/views.py

```diff
@@ -60,14 +69,18 @@ def __init__(self, *args, **kwargs):
 
     def get(self, request):
         """OIDC client authentication initialization HTTP endpoint"""
+        state = get_random_string(import_from_settings('OIDC_STATE_SIZE', 32))
```

Comment:
Although I am in favor of adding this to the request, we should keep that it's recommended and not required to have the `state` in the request.

nit: maybe you could implement this feature as optional, defaulting to True. You can also safely ignore this comment.